app/views/certificates.html

```diff
@@ -122,6 +122,8 @@
 
               <!-- Server Certificate Alerts -->
               <alert ng-repeat="alert in alerts.serverKey" type="alert.type" style="margin: 20px 0px 0px 0px;">{{alert.msg}}</alert>
+              <h4><strong>Enter a passphrase (Optional)</strong></h4>
+              <input class="form-control" placeholder="Enter a passphrase here if the key requires one." type="text" ng-model="serverPassphrase" />
```

Comment:
I think that this should be a password input type. So the password into in plain text. Also, it may be better if it has a fixed width so it doesn't become so long.